Minio/Helper/Utils.cs

```diff
@@ -196,6 +197,21 @@ internal static bool IsSupersetOf(IList<string> l1, IList<string> l2)
         return !l2.Except(l1).Any();
     }
 
+    public static Task ForEachAsync<T>(this IEnumerable<T> source, Func<T, Task> body)
+    {
+        var MaxNoOfParallelProcesses = 4;
```

Comment:
`var maxNoOfParallelProcesses = 4;` non capital since it is a local var

Reply:
Done